packages/graphback-cli/src/helpers/common.ts

```diff
@@ -1,12 +1,27 @@
-import { accessSync } from 'fs'
+import { accessSync, readFileSync } from 'fs'
+import { join } from 'path'
 import { logError } from '../utils'
 
 
-export const checkDirectory = (): void => {
-  try{
-    accessSync(`${process.cwd()}/model`)
-  } catch(err) {
-    logError(`model directory not found. Make you sure you are in the root of your project.`)
+export const checkDirectory = (): string => {
+
+  let pathForModel
+
+  try {
+
+    const configPath = `${process.cwd()}/graphback.json`
+
+    const { paths } = JSON.parse(readFileSync(configPath, "utf8"))
+  
+    pathForModel = join(process.cwd(), paths.model)
+
+    accessSync(`${pathForModel}`)
+
+  } catch (err) {
+    logError(`Model not found, make sure you are in root directory of your project and that you have
```

Comment:
Not issue with your code itself but it can be improved. 
Code above can fail in 2 places:

- on reading config - then we need to say we have a config issue
- on reading graphql files in the model